csharp/ql/src/Documentation/Documentation.qll

```diff
@@ -1,10 +1,6 @@
 import csharp
 
-class SourceDeclaration extends Declaration {
-  SourceDeclaration() {
-    this.isSourceDeclaration()
-  }
-}
+class SourceDeclaration extends Declaration { SourceDeclaration() { this.isSourceDeclaration() } }
```

Comment:
@aschackmull Personally I don't think we should inline class bodies like this, preferring instead
```
  class SourceDeclaration extends Declaration {
    SourceDeclaration() { this.isSourceDeclaration() }
  }
```

Reply:
Hmm, yes, I think I agree.  That might be a good change to make.